packages/data-sdk/src/App.ts

```diff
@@ -387,6 +388,11 @@ export class App {
     });
   }
 
+  static async disableAnalyticsBottomBar() {
+    this.sendAppMessage({
+      type: "hide_analytics_date_picker",
+    });
+  }
```

Comment:
at some point we probably need to figure out a more comprehensive way for the module and _host_ to handshake some _version_ of the protocol; i suppose _usually_ the host ignores messages it doesn't _understand_ or _support_ but… rolling these out in _pairs_ between the `data-sdk` and the support on the `telemetry` deployment side feels… yucks!

Reply:
yea, i dunno yet. but definitely something I need to think about more.